assets/js/admin.js

```diff
@@ -139,42 +137,62 @@ jQuery(document).ready(function($){
    * @since 1.7.0
    */
   //Display controls
-  $('#cf-delete-all-form-entries').click(function(e) {
+  $('#caldera-forms-delete-all-form-entries').click(function(e) {
     e.preventDefault();
-    $('#cf-confirm-delete-all-form-entries').slideToggle("fast");
+    $('#caldera-forms-confirm-delete-all-form-entries').slideToggle("fast");
   });
   //No clicked
-  $('#cf-no-confirm-delete-all-form-entries').click(function(e) {
+  $('#caldera-forms-no-confirm-delete-all-form-entries').click(function(e) {
     e.preventDefault();
-    $('#cf-confirm-delete-all-form-entries').slideToggle("fast");
+    $('#caldera-forms-confirm-delete-all-form-entries').slideToggle("fast");
   });
   //Yes clicked
-  $('#cf-yes-confirm-delete-all-form-entries').click(function(e) {
+  $('#caldera-forms-yes-confirm-delete-all-form-entries').click(function(e) {
     e.preventDefault();
-    var formId = $( this ).attr('data-cf-form-id');
-    $.post(
-      adminAJAX,
-      {
-        'action': 'cf_delete_all_form_entries',
-        'formId': formId
-      },
-      function(response) {
-
-        if( response === "No-entries") {
-          $('#label-cf-delete-all-entries').append("<div class='NoDeleted'>" + noDeletedText + "</div>");
-          setTimeout(function() {
-            $('.NoDeleted').remove();
+
+    var url = CF_ADMIN.rest.delete_entries;
+
+    var $spinner = jQuery( '#caldera-forms-delete-entries-spinner' );
+    $spinner.css({
+      visibility: 'visible',
+      float:'none'
+    });
+
+    jQuery.get(url, function (r) {
```

Comment:
@New0  Please use wp.apiRequest here. It's a wrapper over jQuery.ajax, so easy change.

My reason is that it autosets auth headers with nonce.

Not, you must enqueue core script with handle`wp-api-request` You can do that here: https://github.com/CalderaWP/Caldera-Forms/blob/develop/classes/admin/assets.php#L37

Example: https://github.com/CalderaWP/Caldera-Forms/blob/develop/clients/blocks/store.js#L89-L96